recipe/bld.bat

```diff
@@ -126,7 +126,7 @@ if "%PKG_NAME%" == "libtorch" (
 
   :: NOTE: Passing --cmake is necessary here since the torch frontend has it's
   :: own cmake files that it needs to generate
-  %PYTHON% setup.py clean
+  cmake -DPython_EXECUTABLE="%PYTHON%" --build build --target clean
```

Comment:
not sure about this. you seem to be mixing setting a configuration variable and running a build step. Try removing `-DPython_EXECUTABLE="%PYTHON%"`. This would be a change from using pytorch's clean to using cmake's build stage clean. (i.e., it would clean build artifacts without regenerating a build system)

Reply:
I brought that over straight from the conda-forge PR. I'll try removing the`-DPython_EXECUTABLE ` and make sure it still solves the problem of the 3.12 not getting fully built. 

The `setup.py clean` wasn't ever actually doing anything since it relies on the .gitignore file being present (it's not present in the tarball we are pulling down" 

https://github.com/pytorch/pytorch/blob/main/setup.py#L874-L892